lib/linters/enhanced_idv_events_linter.rb

```diff
@@ -0,0 +1,220 @@
+module RuboCop
+  module Cop
+    module IdentityIdp
+      class EnhancedIdvEventsLinter < RuboCop::Cop::Base
```

Comment:
Honestly, here be dragons. But analytics_events.rb is long enough that automating large edits to it was worth the trouble.